docs/guide/installation/index.rst

```diff
@@ -2,59 +2,63 @@
 Installation
 ============
 
-SunPy relies on and enables the use of the wider ecosystem of scientific Python packages for solar physics.
-Therefore a working SunPy installation is more about installing the scientific Python ecosystem than SunPy itself.
-SunPy v1.X or greater is compatible with Python 3.6+.
-If you still want a Python 2 version, you have to install SunPy v0.9.X.
-
-If you are new to Python and scientific Python then continue to follow this guide to get setup with the whole environment.
+If you are new to Python and scientific Python then see the :ref:`main-install` section to get setup with the whole environment.
 If you already have a working Python / Scientific Python environment then you can skip to the :ref:`advanced-install` section.
 
+Requirements
+------------
+
+- python >= 3.6
+- astropy >= 3.2
+
+sunpy v1.X or greater is compatible with Python 3.6+.
+If you need a Python 2 compatible version, you have to install SunPy v0.9.X.
```

Comment:
Should we advertise this? We don't support it anymore.

Reply:
Good point, especially since 2.0 will be a second python 3+ LTS